index.js

```diff
@@ -1,30 +1,28 @@
-const express = require('express');
-const app = express();
-
-var path = require('path');
+var app = require('express')();
```

Comment:
Move all Express Rest  and socketIO function handling to separated classes and files for better organizing.